hydroshare/settings.py

```diff
@@ -313,8 +317,15 @@
     "hs_rest_api",
     "hs_dictionary",
     "security",
+    "sass_processor",
```

Comment:
I like this, but one thing that I'm not seeing is: sourcemaps. Did you get that to work? Did you have to do anything special to do so?

Reply:
you know...I see source maps genereted in the static dirs, but I didn't bother to test them. I'll take a poke at that.

Reply:
they are generated - but the paths look wrong. It looks like *maybe* this will be fixed in the next release: https://github.com/jrief/django-sass-processor/pull/96